### ISSUE TITLE
[BUG] curand_kernel.h: No such file or directory

### ISSUE THREAD

**Issue by BinhMinhs10** [microsoft/DeepSpeed#3411]
**Describe the bug**
Installed CUDA version 11.2 does not match the version torch was compiled with 11.7 but since the APIs are compatible, accepting this combination
Using /home/jovyan/.cache/torch_extensions/py38_cu117 as PyTorch extensions root...

In file included from /home/jovyan/vol-1/env/venv/lib/python3.8/site-packages/deepspeed/ops/csrc/includes/cpu_adam.h:19,
                 from /home/jovyan/vol-1/env/venv/lib/python3.8/site-packages/deepspeed/ops/csrc/adam/cpu_adam.cpp:6:

In file included from /opt/conda/lib/python3.8/site-packages/deepspeed/ops/csrc/common/custom_cuda_kernel.cu:6:
/opt/conda/lib/python3.8/site-packages/deepspeed/ops/csrc/includes/custom_cuda_layers.h:12:10: fatal error: curand_kernel.h: No such file or directory
   12 | #include <curand_kernel.h>
      |          ^~~~~~~~~~~~~~~~~
compilation terminated.
ninja: build stopped: subcommand failed.

**To Reproduce**
accelerate launch --config_file ds_zero3_cpu.yaml examples/peft_lora_seq2seq_accelerate_ds_zero3_offload.py
Follow : https://github.com/huggingface/peft

**Comment by BinhMinhs10** [comment]
I have solved the problem by installing cuda-toolkit 11.7 and resetting LD_LIBRARY_PATH to /opt/conda/lib64

**Comment by Kin-Zhang** [comment]
> I have solved the problem by installing cuda-toolkit 11.7 and resetting LD_LIBRARY_PATH to /opt/conda/lib64

Add some details: (previously I only installed cuda not cuda-toolkit 

```bash
conda install nvidia/label/cuda-12.1.0::cuda
conda install nvidia/label/cuda-12.1.0::cuda-toolkit
```

and also export lib: `export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/home/$USER/miniforge3/lib`
